src/test/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SubmitYourAppealToCcdCaseDataDeserializerTest.java

```diff
@@ -426,21 +426,37 @@ public void syaWithRepNameUndefinedWillMapToNull() {
     }
 
     @Test
-    public void syaWithAppellantNameSetsWorkAllocationFields() {
+    public void givenSyaWithAppellantName_shouldSetCaseAccessManagementFields() {
         SyaCaseWrapper syaCaseWrapper = ALL_DETAILS.getDeserializeMessage();
         syaCaseWrapper.getAppellant().setFirstName("George");
         syaCaseWrapper.getAppellant().setLastName("Foreman");
+
         SscsCaseData caseData = convertSyaToCcdCaseData(syaCaseWrapper, regionalProcessingCenter.getName(), regionalProcessingCenter, true);
-        assertEquals("George Foreman", caseData.getWorkAllocationFields().getCaseNameHmctsInternal());
-        assertEquals("George Foreman", caseData.getWorkAllocationFields().getCaseNameHmctsRestricted());
-        assertEquals("George Foreman", caseData.getWorkAllocationFields().getCaseNamePublic());
 
-        assertEquals("personalIndependencePayment", caseData.getWorkAllocationFields().getCaseAccessCategory());
-        assertEquals("DWP", caseData.getWorkAllocationFields().getOgdType());
-        assertEquals("Personal Independence Payment", caseData.getWorkAllocationFields().getCaseManagementCategory().getValue().getLabel());
-        assertEquals("PIP", caseData.getWorkAllocationFields().getCaseManagementCategory().getValue().getCode());
+        assertEquals("George Foreman", caseData.getCaseAccessManagementFields().getCaseNameHmctsInternal());
+        assertEquals("George Foreman", caseData.getCaseAccessManagementFields().getCaseNameHmctsRestricted());
+        assertEquals("George Foreman", caseData.getCaseAccessManagementFields().getCaseNamePublic());
```

Comment:
George Foreman constant?

Reply:
I like the way this flows as you can see the value in the assertion rather than having to look it up but will change it.

Reply:
Fine :)